### DISCUSSION TITLE
Can I Build on windows?

### DISCUSSION THREAD

**Discussion by crypto-keys-unlocked** [openfheorg/openfhe-development#523] [Q&A]
Hi I was trying to build the library on windows but getting some errors. Is this suitable to build on windows or only Linux is supported/

Errors:
![image](https://github.com/openfheorg/openfhe-development/assets/122903825/f47a20da-4204-4d23-826d-a6e4ac2fef7c)

**Comment by yspolyakov** [comment]
Please ask this question at https://openfhe.discourse.group/ if you are still having this issue.
